main/src/library/Reducible.flix

```diff
@@ -38,34 +38,48 @@ pub lawless class Reducible[t: Type -> Type] {
     ///
     /// Left-associative reduction on `t` using `f`.
     ///
-    pub def reduceLeft(f: (a, a) -> a & ef, t: t[a]): a = ???
+    pub def reduceLeft(f: (a, a) -> a & ef, t: t[a]): a & ef =
+        Reducible.reduceLeftTo(f, identity, t)
```

Comment:
You can just call Reducible.reduceLeft directly. (same comment applies elsewhere).

Reply:
Whoops. I am not in the file I thought :)